### PR TITLE
New package: iamb-0.0.10

### PR DESCRIPTION
**Pull request by ninewise**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**. Well, mostly. It's still 0.0.10, but it has had releases since march 2023.

### CODE REVIEW

File: srcpkgs/iamb/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'iamb'
+pkgname=iamb
+version=0.0.10
+revision=1
+build_style=cargo
+configure_args="--no-default-features --features=native-tls"
```
Comment (tranzystorekk) [inline srcpkgs/iamb/template:6]: Also enable the `desktop` feature

Comment (ninewise) [reply srcpkgs/iamb/template]: Indeed; it didn't need that on 0.0.9.

File: srcpkgs/iamb/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+distfiles="https://github.com/ulyssa/iamb/archive/refs/tags/v${version}.tar.gz"
+checksum=f628cfbd9eba9e8881902b970e9432fec815044ec9bea901a8562ea3ef8f4615
+
+post_install() {
+	vinstall iamb.desktop 644 usr/share/applications
+	vinstall config.example.toml 644 usr/share/examples/iamb config.toml
+	vinstall docs/iamb.svg 644 usr/share/icons/hicolor/scalable/apps
+	vman docs/iamb.1
+	vman docs/iamb.5
+}
```
Comment (tranzystorekk) [inline srcpkgs/iamb/template:22]: We should also install the metainfo file for completeness

File: srcpkgs/iamb/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+version=0.0.10
+revision=1
+build_style=cargo
+configure_args="--no-default-features --features=native-tls,desktop"
+hostmakedepends="pkg-config"
+makedepends="openssl-devel sqlite-devel oniguruma-devel"
+short_desc="Terminal-based client for Matrix for the Vim addict"
+maintainer="Felix Van der Jeugt <felixvdj+github@posteo.be>"
+license="Apache-2.0"
+homepage="https://iamb.chat"
```
Comment (tranzystorekk) [inline srcpkgs/iamb/template:12]:
```suggestion
homepage="https://iamb.chat"
changelog="https://github.com/ulyssa/iamb/releases"
```

**Comment by Bnyro** [comment]
Duplicate of https://github.com/void-linux/void-packages/pull/49652

**Pull request closed without merging**